### DISCUSSION TITLE
Language change of the embedded wiki in "OSU!Lazer".

### DISCUSSION THREAD

**Discussion by gi-b716** [ppy/osu#27942] [Ideas] [answered]
(Translated from Chinese using DeepL, please bear with me if there are any grammatical errors.)
There is a language switching button at the top right of the web version of osu!wiki, but he disappeared from the inline wiki of OSU!Lazer. Suggest that the embedded Wiki in OSU!Lazer could provide buttons to change the language, or automatically adjust to the client's language instead of always English.

**Comment by Walavouchey** [comment, marked as answer]
this has already been reported in https://github.com/ppy/osu/discussions/22063 and an implementation has been attempted at https://github.com/ppy/osu/pull/23296, but it is stuck in review
